tests/subdirectory/default.nix

```diff
@@ -0,0 +1,3 @@
+{ poetry2nix }:
+
+poetry2nix.mkPoetryEnv { projectDir = ./.; }
```

Comment:
Would an import test (see for example `tests/jq/default.nix`) be useful to strengthen the assertion this test is making?

Reply:
The test fails at build time without subdirectory support, but sure, added a runtime import test.